src/coreclr/nativeaot/Runtime/unix/UnwindHelpers.cpp

```diff
@@ -883,3 +883,29 @@ bool UnwindHelpers::GetUnwindProcInfo(PCODE pc, UnwindInfoSections &uwInfoSectio
     uc.getInfo(procInfo);
     return true;
 }
+
+#if defined(TARGET_APPLE)
+bool _dyld_find_unwind_sections(void* addr, dyld_unwind_sections* info)
```

Comment:
```suggestion
// Apple considers _dyld_find_unwind_sections to be private API that cannot be used
// by apps submitted to App Store and TestFlight, both for iOS-like and macOS platforms.
// We reimplement it using public API surface.
//
// Ref: https://github.com/apple-oss-distributions/dyld/blob/ce1cc2088ef390df1c48a1648075bbd51c5bbc6a/dyld/DyldAPIs.cpp#L1859-L1884
// Ref: https://github.com/llvm/llvm-project/blob/c37145cab12168798a603e22af6b6bf6f606b705/libunwind/src/AddressSpace.hpp#L67-L93
bool _dyld_find_unwind_sections(void* addr, dyld_unwind_sections* info)
```

Sounds good?

Reply:
I would avoid reference to apple-oss-distributions, so that this is not considered a modified version of that code. apple-oss-distributions comes with complicated license.

The llvm reference should be enough. In fact, this is almost exact copy of the llvm code.